seerep_srv/seerep_server/src/fb_meta_operations.cpp

```diff
@@ -47,55 +47,64 @@ grpc::Status FbMetaOperations::GetProjects(grpc::ServerContext* context,
+    auto geoCordsOffset = seerep::fb::CreateGeodeticCoordinatesDirect(
+        fbb, projectInfo.geodetCoords.coordinateSystem.c_str(), projectInfo.geodetCoords.ellipsoid.c_str(),
+        projectInfo.geodetCoords.longitude, projectInfo.geodetCoords.latitude, projectInfo.geodetCoords.altitude);
+
+    projectInfosVector.push_back(seerep::fb::CreateProjectInfoDirect(
+        fbb, projectInfo.name.c_str(), boost::lexical_cast<std::string>(projectInfo.uuid).c_str(),
+        projectInfo.frameId.c_str(), geoCordsOffset, projectInfo.version.c_str()));
   }
-  auto vectorOffset = builder.CreateVector(projectInfosVector);
-  auto projectInfosOffset = seerep::fb::CreateProjectInfos(builder, vectorOffset);
-  builder.Finish(projectInfosOffset);
-  *response = builder.ReleaseMessage<seerep::fb::ProjectInfos>();
+
+  auto projectInfosOffset = seerep::fb::CreateProjectInfos(fbb, fbb.CreateVector(projectInfosVector));
+  fbb.Finish(projectInfosOffset);
+
+  *response = fbb.ReleaseMessage<seerep::fb::ProjectInfos>();
   assert(response->Verify());
 
   return grpc::Status::OK;
 }
 
 grpc::Status FbMetaOperations::LoadProjects(grpc::ServerContext* context,
                                             const flatbuffers::grpc::Message<seerep::fb::Empty>* request,
-                                            flatbuffers::grpc::Message<seerep::fb::Empty>* response)
+                                            flatbuffers::grpc::Message<seerep::fb::ProjectInfos>* response)
 {
-  (void)context;  // ignore that variable without causing warnings
-  (void)request;  // ignore that variable without causing warnings
-  BOOST_LOG_SEV(m_logger, boost::log::trivial::severity_level::info)
-      << "loading all projects in the working directory... ";
+  (void)context;
+  (void)request;
+
+  BOOST_LOG_SEV(m_logger, boost::log::trivial::severity_level::debug)
+      << "[FB] loading new projects in the working directory ... ";
+
+  flatbuffers::grpc::MessageBuilder fbb;
+  std::vector<seerep_core_msgs::ProjectInfo> projectInfos = seerepCore->loadProjectsInFolder();
 
   try
   {
-    seerepCore->loadProjectsInFolder();
+    std::vector<flatbuffers::Offset<seerep::fb::ProjectInfo>> projectInfosVector;
+    for (auto projectInfo : projectInfos)
+    {
+      auto geoCordsOffset = seerep::fb::CreateGeodeticCoordinatesDirect(
+          fbb, projectInfo.geodetCoords.coordinateSystem.c_str(), projectInfo.geodetCoords.ellipsoid.c_str(),
+          projectInfo.geodetCoords.longitude, projectInfo.geodetCoords.latitude, projectInfo.geodetCoords.altitude);
+
+      projectInfosVector.push_back(seerep::fb::CreateProjectInfoDirect(
+          fbb, projectInfo.name.c_str(), boost::lexical_cast<std::string>(projectInfo.uuid).c_str(),
+          projectInfo.frameId.c_str(), geoCordsOffset, projectInfo.version.c_str()));
+    }
+    auto projectInfosOffset = seerep::fb::CreateProjectInfos(fbb, fbb.CreateVector(projectInfosVector));
+    fbb.Finish(projectInfosOffset);
```

Comment:
Yes we should do that, it can also be used for `getProjects` function https://github.com/agri-gaia/seerep/blob/e39f3c7a071cf9e0318d6174166b261b53c9886e/seerep_srv/seerep_server/src/fb_meta_operations.cpp#L58-L72